.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# Description
+
+<!-- Please use this template to file a pull request for opta and provide as much info as possible. 
+We'll be making script based off this structure for releases and will be annoyed if it is not followed.
+Thank you!-->
+
+# Safety checklist
+* [ ] This change is backwards compatible and safe to apply by existing users
+* [ ] This change will NOT lead to data loss
+* [ ] This change will NOT lead to downtime who already has an env/service setup
+
+## Besides the unit tests (which will now run), how else has this been tests?
```

Comment:
I think you can just write `How has this change been tested, beside unit tests?`